FpsSet.py

```diff
@@ -25,7 +25,7 @@ class FpsSet:
     relevant_fields_dict: a dictionary mapping the JSON field equivalents
     of each field to their value within the object. 
   """
-    def __init__(self, ccTLDs, primary, associated_sites, service_sites):
+    def __init__(self, ccTLDs, primary, associated_sites=None, service_sites=None):
```

Comment:
```suggestion
    def __init__(self, *, ccTLDs, primary, associated_sites=None, service_sites=None):
```

Optional: You could enforce usage of keyword arguments (rather than positional args) by adding `*, ` here.